chain-abci/tests/abci_app.rs

```diff
@@ -800,12 +784,15 @@ fn block_commit(app: &mut ChainNodeApp<MockClient>, tx: TxAux, block_height: i64
 
 fn get_account(account_address: &RedeemAddress, app: &ChainNodeApp<MockClient>) -> StakedState {
     println!(
-        "uncommitted root hash: {:?}",
-        app.uncommitted_account_root_hash
+        "uncommitted root hash: {}",
+        hex::encode(app.uncommitted_account_root_hash)
     );
     let account_key = to_stake_key(&StakedStateAddress::from(*account_address));
     let state = app.last_state.clone().expect("app state");
-    println!("committed root hash: {:?}", &state.last_account_root_hash);
+    println!(
```

Comment:
debug!

Reply:
this is in tests, so it won't be printed out unless the test fails

Reply:
so `println!` is OK